pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -397,11 +397,6 @@ func (c *Controller) runReconcileShootFlow(o *operation.Operation) *gardencorev1
 			Fn:           flow.TaskFn(botanist.WaitUntilExtensionResourcesDeleted).SkipIf(o.Shoot.HibernationEnabled),
 			Dependencies: flow.NewTaskIDs(deleteStaleExtensionResources),
 		})
-		_ = g.Add(flow.Task{
```

Comment:
What will remove the `garden.sapcloud.io/createdBy` annotation from existing clusters?

Reply:
Is it necessary to remove it? We can simply keep it for old/existing resources and save the code.

Reply:
It is not required to be removed, but we are doing it for other annotations/labels thus I do not see a reason why it should be different for this one. If there is a reason, I am fine to keep it.

Reply:
I added deletion for the createdBy annotation, thanks for the catch.